File: Weekend Exercises/Python Version/task_ten.py
number = int(input("Enter a number: "))

new_number = str(number)

reversed_number = ""

for num in new_number:

	reversed_number = num + reversed_number

if (str(number) == reversed_number):

	print(f"{number} is palindrome")

else:
	print(f"{number} is not palindrome")

File: Weekend Exercises/Java Version/TaskTen.java
import java.util.Scanner;
public class TaskTen{

	public static void main (String [] args){

		Scanner input = new Scanner (System.in);

		System.out.println("Enter a number: ");
		int number = input.nextInt();

		String numberToString = String.valueOf(number);

		String reverseString = "";

		for (int index = 0; index < numberToString.length(); index++){
			reverseString = numberToString.charAt(index) + reverseString;
		}

		int reverseInteger = Integer.valueOf(reverseString);

		if (reverseInteger == number){
			System.out.printf("%d is a palindrome%n", number);
		}
		else{
			System.out.printf("%d is not palindrome%n", number);
		}

	}
}